test/cache.bats

```diff
@@ -57,14 +56,15 @@ setup() {
   export RUBY_BUILD_SKIP_MIRROR=
   local checksum="ba988b1bb4250dee0b9dd3d4d722f9c64b2bacfc805d1b6eba7426bda72dd3c5"
 
-  stub shasum true "echo invalid" "echo $checksum"
+  stub shasum true true "echo invalid" true "echo $checksum"
```

Comment:
It's hard to follow all these arguments to stubbed `shasum`, and the extra `true`s only make it worse. Can we avoid adding them? What causes the necessity for extra arguments?